package/Dockerfile

```diff
@@ -1,34 +1,39 @@
-FROM nginx:1.21.6-alpine
+FROM nginx:1.24.0-alpine
+
+ENV DOCKER_VERSION=23.0.3
+ENV PLUGINS_VERSION=v1.2.0
+ENV ETCD_VERSION=v3.4.25
+ENV CRIDOCKERD_VERSION=0.3.1
+ENV RANCHER_CONFD_VERSION=v0.16.4
+ENV KUBECTL_VERSION=v1.27.1
```

Comment:
Is this the right version to maintain compatibility between k8s versions? Similar to shell repository I believe.

Reply:
The version in `master` is `v1.18.2`, which supports apiservers: `v1.16.*`, `v1.17.*` and `v1.18.*` - all of which are EOL.
The proposed changes [support apiservers](https://kubernetes.io/releases/version-skew-policy/#kubectl): `v1.25.*`, `v1.24.*` and `v1.23.*`, which aligns with current Rancher Manager [supported versions](https://www.suse.com/suse-rancher/support-matrix/all-supported-versions/rancher-v2-7-5/).

Reply:
Changed the version of kubectl to `v1.24.15`, which is the lowest supported minor - and aligns with apiserver versions above.